src/libtools/graph.cc

```diff
@@ -106,7 +107,10 @@ void Graph::setNode(std::pair<int, GraphNode> p_gn) {
   setNode(p_gn.first, p_gn.second);
 }
 
-GraphNode Graph::getNode(int vert) { return nodes_[vert]; }
+GraphNode Graph::getNode(int vert) { 
+  assert(nodes_.count(vert));
```

Comment:
Keep in mind that `assert` works only when `NDEBUG` macro is not defined.

Reply:
👍